src/system-health/health_checker/hardware_checker.py

```diff
@@ -239,6 +239,18 @@ def _check_psu_status(self, config):
                                                                                                                voltage_min_th,
                                                                                                                voltage_max_th))
                         continue
+
+            if not self._ignore_check(config.ignore_devices, 'psu', name, 'power_threshold'):
+                power_overload = data_dict.get('power_overload', None)
+                if power_overload == 'True':
+                    try:
+                        power = data_dict['power']
+                        power_critical_threshold = data_dict['power_critical_threshold']
```

Comment:
what if 'power_critical_threshold' is not applicable or supported for a platform

Reply:
there will be an `N/A` in the entry. it is pushed by the PSU daemon.